packages/contracts-rfq/contracts/interfaces/IFastBridgeV2.sol

```diff
@@ -20,4 +20,15 @@ interface IFastBridgeV2 is IFastBridge {
     /// @param request The encoded bridge transaction to claim on origin chain
     function claim(bytes memory request) external;
 
+    /// @notice Checks if a transaction has been relayed
```

Comment:
Needs `forge fmt` + in general we try to define the structs at the top of the contract, prior to the functions. In hindsight, both of these should've been flagged by CI workflows - I will make a ticket to update these with a reasonable settings (e.g. older `FastBridge` has a ton of linter warnings that might as well be ignored at this point)